client/utils.js

```diff
@@ -34,6 +34,10 @@ export function clamp(value, min, max) {
   return Math.max(min, Math.min(max, value));
 }
 
+export function sign(value) {
+  return value < 0 ? -1 : value > 0 ? 1 : 0;
```

Comment:
we can use [Math.sign](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/sign)

Reply:
Ah thanks, didn't know about that